src/test/java/org/jenkinsci/plugins/docker/workflow/client/DockerClientTest.java

```diff
@@ -76,6 +76,26 @@ public void test_run() throws IOException, InterruptedException {
     }
 
     @Test
+    public void test_run_without_user() throws IOException, InterruptedException {
+        EnvVars launchEnv = newLaunchEnv();
+        String containerId =
+                dockerClient.run(launchEnv, "learn/tutorial", null, null, Collections.<String, String>emptyMap(), Collections.<String>emptyList(), new EnvVars(),
+                        null, "echo", "hello world");
+        Assert.assertEquals(64, containerId.length());
+        ContainerRecord containerRecord = dockerClient.getContainerRecord(launchEnv, containerId);
+        Assert.assertEquals(dockerClient.inspect(launchEnv, "learn/tutorial", ".Id"), containerRecord.getImageId());
+        Assert.assertTrue(containerRecord.getContainerName().length() > 0);
+        Assert.assertTrue(containerRecord.getHost().length() > 0);
+        Assert.assertTrue(containerRecord.getCreated() > 1000000000000L);
+        Assert.assertEquals(Collections.<String>emptyList(), dockerClient.getVolumes(launchEnv, containerId));
```

Comment:
The original intent was to test that not passing in a user would not define a user, but the ContainerRecord doesn't provide a user field so I settled for just ensuring that when passing a null value into the user field you still received a valid ContainerRecord.  I can remove this if you want.